index.html

```diff
@@ -31,12 +31,133 @@ <h1 class="about-title">I’m Martin.Glad to see you!</h1>
         </p>
         <p class="connection">LET’S CONNECT</p>
         <ul>
-            <li><a href="#"><img src="image/Vector (15).png" alt="twitter"></a></li>
+            <li ><a href="#"><img src="image/Vector (15).png" alt="twitter"></a></li>
             <li><a href="#"><img src="image/Linkedin icon.png" alt="linkedin"></a></li>
             <li><a href="#"><img src="image/Vector (11).png" alt="mail"></a></li>
             <li><a href="#"><img src="image/Vector (12).png" alt="github"></a></li>
             <li><a href="#"><img src="image/Vector (13).png" alt="peace"></a></li>
         </ul>
     </section>
+    <!-- WORK SECTION -->
+    <section class="work">
+        <!-- CARD 1 -->
+        <div class="card-1">
+            <img src="image/Snapshoot Portfolio.png" alt="image1" >
+            <h1 class="tonic">Tonic</h1>
+            <div class="canop">
+                <p class="can-info">CANOPY</p>
+                <div class="point"></div>
+                <p class="back">Back End Dev</p>
+                <div class="point"></div>
+                <p class="back">2015</p>
+            </div>
+            <div class="languages">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>JavaScript</p>
+            </div>
+            <button class="see">See project</button>
+        </div>
```

Comment:
Great job on wrapping all the projects within a section! However, there are a few adjustments that need to be made:

- [ ] The content of each project should be aligned using Flexbox, as per the project requirements. Currently, it seems that Flexbox is not being utilized. I recommend using flexbox to align the elements, as it provides an easier way to layout elements using properties like `display: flex`, `justify-content`, and `gap`. Please update your code to incorporate Flexbox for better alignment.
- [ ] In terms of HTML structure, it would be more semantically appropriate to use the `<article>` tag instead of `<div>` to represent each project card. The `<article>` tag conveys the purpose of containing an independent, self-contained piece of content. Please make the necessary changes to your code to use the `<article>` tag for each project.
- [ ] The spacing between the first project and the previous section is too large compared to the design. Please adjust the spacing to match the design specifications.
![image](https://github.com/mar4444/re-portfolio/assets/55840999/9b46f726-84f2-43e0-8fc0-a0b94ebbf5d2)
- [ ] Additionally, the gap between two project cards is currently larger than the design. Please follow the design and reduce the space between the project cards to align with the design guidelines.
![image](https://github.com/mar4444/re-portfolio/assets/55840999/8596fb0a-3493-4234-b30e-c1af7eb4f9cc)
Please address these adjustments to ensure that your projects section is visually appealing and aligns with the design requirements. Keep up the good work! 👏🌟